### ISSUE TITLE
Predicted move & accuracy in percentage

### ISSUE THREAD

**Issue by Nordlandia** [cutechess/cutechess#239]
Thanks a lot for releasing v1.0.0 🌻

Pondermove & ponderhit work only if permanent brain is activated. Ponder comes at 50% CPU penalty for each engine, on single CPU ship. Thus many engine testers avoid using ponder. 

Predicted move accuracy functions same way but it doesn't require ponder to be enabled. 

This is meant as an improvement. 

This feature is implemented in Fritz GUI, - in ongoing engine matches, one can see the percentage. 

![123](https://user-images.githubusercontent.com/26534454/28472501-16a58896-6e41-11e7-8fb0-8dbc9c0f9ae4.png)

**Comment by Nordlandia** [comment]
Alternatively ponder move & percentage should work if ponder is not enabled.

**Comment by ilaripih** [comment]
Yes, I can see that this would be useful. Most UCI engines print the ponder move even when pondering is disabled. And if they don't or if the engine uses Xboard protocol, we can get the predicted move from the last PV line.

**Comment by ilaripih** [comment]
Pondermove and ponderhit are now shown for UCI engines even when pondering is disabled.

**Comment by Nordlandia** [comment]
I tried update MSYS2 installer and new compiled GUI don't display 😢

**Comment by ilaripih** [comment]
The changes were in cutechess-lib. It's possible that you have to completely recompile it before linking to cutechess-gui.

**Comment by Nordlandia** [comment]
I'm following _making a release_ recipe, which turns out don't work (doesn't include latest implemention)

Recompile msys64 content is new to me.

https://github.com/cutechess/cutechess/wiki/Making-a-release

**Comment by ilaripih** [comment]
Let's reopen this, it looks like my implementation of predicted move is not correct. I tested with Stockfish which always sends the ponder move, but other engines may not do that if pondering is disabled.

**Comment by Nordlandia** [comment]
CuteChess report pondermove but unfortunately it's the previous move. It looks like the tab don't update once one side has made it's move. It report previous move and sometimes get stuck on that for a while.
